build/index.html

```diff
@@ -3,6 +3,7 @@
     <link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
We were previously importing the `style.css` like this. `style.css` should now be a scss file `style.scss` and should be in the `styles` folder. 

`main.scss` should `@import "styles"`